tools/api-proxy/secureapi-proxy.js

```diff
@@ -8,36 +8,110 @@
  * @param env
  * @returns {Promise<Response>}
  */
-
+/* eslint-disable */
```

Comment:
Why disable linter?

Reply:
@sdmcraft  this code is actually managed in CloudFlare worker and initially wrote on that console. 
Its there since there since start. Let me remove this. It is not required anymore and fix any linting error if present.

Reply:
@sdmcraft updated